src/containers/Stories/index.jsx

```diff
@@ -2,8 +2,13 @@ import React from "react";
 import { Container } from "react-bootstrap";
 import SingleStoryCard from "../../components/SingleStoryCard";
 import LoadingPage from "../../components/LoadingPage";
+import i18next from "i18next";
 
 const Stories = () => {
+  const en = "10221";
```

Comment:
these consts should be all in caps since these won't be changing

Reply:
and lets make it more specific smth like ENGLISH_LANGUAGE

Reply:
Fixed